src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block toolbar %}
+    {% set color_code = 'normal' %}
+    {% set message_count = 0 %}
+    {% set icon %}
+        {{ include('@WebProfiler/Icon/messenger.svg') }}
+        <span class="sf-toolbar-value">{{ message_count }}</span>
+    {% endset %}
+
+    {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: 'messages', status: color_code }) }}
```

Comment:
you could even avoid passing the status explicitly, as you never change it to anything else than `normal` (which is the default status)

Reply:
Removed too!